backend/clubs/views.py

```diff
@@ -3422,7 +3424,8 @@ def get_queryset(self):
         )
 
         if not self.request.user.is_authenticated:
-            return questions.filter(approved=True)
+            # Hide responder if not authenticated
+            return questions.filter(approved=True).extra(select={"responder": "NULL"})
```

Comment:
Not sure if this is best practice but I feel that we should avoid using SQL if possible. Feel free to ignore though as this is slightly more verbose:

```suggestion
            return questions.filter(approved=True).annotate(
    responder=Value(None))
```

Reply:
Annotate seems to [not allow overwriting](https://code.djangoproject.com/ticket/28072) of existing fields so extra seems to be a workaround.